### ISSUE TITLE
Can I add some markers in PanoramaView?

### ISSUE THREAD

**Issue by xioozq** [lightbasenl/react-native-panorama-view#9]
like : change scene,  information markers

**Comment by mimmobordini** [comment]
or maybe something like VRButtons

**Comment by aschakra1987** [comment]
> Do you mean adding controls on top of the PanoramaView @xioozq ?

Yes, is there a way to put controls on top of the PanoramaView at specific points which moves with the image?

**Comment by promit13** [comment]
any solution for this question ?
